Makefile

```diff
@@ -97,11 +97,11 @@ create-install-files:
 create-install-files-dev:
 	mkdir -p ./installfilesdev && \
 	INSTALL_FILES_FOLDER=../../installfilesdev \
-	IMG=$(NS)$(IMAGE_NAME_OPERATOR):$${IMAGE_TAG} \
+	IMG=$(NS)$(IMAGE_NAME_OPERATOR):${IMAGE_TAG} \
 	IMAGE_NAME_INIT_CONTAINER=$(NS)$(IMAGE_NAME_INIT_CONTAINER) \
 	IMAGE_NAME_NODE_AGENT=$(NS)$(IMAGE_NAME_NODE_AGENT) \
-	INIT_CONTAINER_TAG=$${IMAGE_TAG} \
-	NODE_AGENT_TAG=$${IMAGE_TAG} \
+	INIT_CONTAINER_TAG=${IMAGE_TAG} \
+	NODE_AGENT_TAG=${IMAGE_TAG} \

```

Comment:
this did not work either, but the following seems to be doing the trick (still testing):

```
create-install-files:
	. .versions && \
	IMG=$(NS)$(IMAGE_NAME_OPERATOR):$${IMAGE_TAG} \
	IMAGE_NAME_INIT_CONTAINER=$(NS)$(IMAGE_NAME_INIT_CONTAINER) \
	IMAGE_NAME_NODE_AGENT=$(NS)$(IMAGE_NAME_NODE_AGENT) \
	IMAGE_TAG=$${IMAGE_TAG} \
	make -C pkg/operator create-install-files

create-install-files-dev:
	mkdir -p ./installfilesdev && \
	INSTALL_FILES_FOLDER=../../installfilesdev \
	IMG=$(NS)$(IMAGE_NAME_OPERATOR):${IMAGE_TAG} \
	IMAGE_NAME_INIT_CONTAINER=$(NS)$(IMAGE_NAME_INIT_CONTAINER) \
	IMAGE_NAME_NODE_AGENT=$(NS)$(IMAGE_NAME_NODE_AGENT) \
	IMAGE_TAG=${IMAGE_TAG} \
	make -C pkg/operator create-install-files
```

Reply:
and, since we're gonna remove the INIT_CONTAINER_TAG completely, can you replace it with IMAGE_TAG here as well?
```
test: manifests generate fmt vet ## Run tests.
	mkdir -p ${ENVTEST_ASSETS_DIR}
	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); THUNDERNETES_SAMPLE_IMAGE=$(IMAGE_NAME_SAMPLE):$(NETCORE_SAMPLE_TAG) THUNDERNETES_INIT_CONTAINER_IMAGE=$(IMAGE_NAME_INIT_CONTAINER):$(INIT_CONTAINER_TAG) go test -v ./... -coverprofile cover.out
```

pkg/operator/Makefile, line 58

Reply:
Also on previous line, maybe we can change the controller-runtime version, since we upgraded

Reply:
let's use the init_container_tag as implied for now. I've made a change, though oddly I was getting the same results when just using the unified IMAGE_TAG. Perhaps an investigation for a later PR